package.json

```diff
@@ -50,7 +50,7 @@
     "fix:license": "gulp update-license-headers",
     "clean": "rm -rf .go_workspace .tmp coverage dist npm-debug.log",
     "postversion": "node aio/scripts/version.js",
-    "postinstall": "node aio/scripts/version.js && ./node_modules/protractor/bin/webdriver-manager update --gecko=false"
+    "postinstall": "node aio/scripts/version.js && ./node_modules/protractor/bin/webdriver-manager update --gecko=false && go get -u github.com/golangci/golangci-lint/cmd/golangci-lint && echo ${PATH} && echo ${GOPATH} && which golangci-lint"
```

Comment:
This is failing because you're attempting to run a `go` command before `go` has been installed.

`go` is installed at this step (after Travis has run `npm ci`): https://github.com/kubernetes/dashboard/blob/master/.travis.yml#L25

I'd move the `eval "$(gimme 1.11.4)"` to the end of `before_install` and then it should work.